presto-benchmark-runner/src/main/java/com/facebook/presto/benchmark/framework/BenchmarkQuery.java

```diff
@@ -35,12 +38,26 @@ public BenchmarkQuery(
             @ColumnName("query") String query,
             @ColumnName("catalog") String catalog,
             @ColumnName("schema") String schema)
+    {
+        this(querySet, name, query, catalog, schema, Optional.empty(), Optional.empty());
+    }
+
+    public BenchmarkQuery(
+            String querySet,
```

Comment:
The table benchmark_queries has this field . So we will probably need it ?

Reply:
We should not be introducing code that are dead on arrival. They're considered bad code quality as they might remain as unused code permanently dead and forgotten, since many time what will happen are different from what is planned, or requirements and goals might be changing. We should only introduce them when we have to.